### PR TITLE
#1353 & #1354 refactor: idea handler & idea card

### PR DESCRIPTION
**Pull request by rz-p**

### CODE REVIEW

**Comment by rz-p** [comment]
Conditionals  are **NOT** working yet, but the cards are technically functional (CRUD works). 

Not sure what to do with `IdeaDetails` as it is pretty much the same as `IdeaCard`

File: hms-app/src/components/card-details/IdeaDetails.tsx [hunk 1]
```diff
@@ -132,6 +132,212 @@ export default function IdeaDetails(props: IProps) {
     </div>
   ))
 
+
+  // Idea Card Functions
+  const ideaName = () => {
```
Comment (LetsPoke) [inline hms-app/src/components/card-details/IdeaDetails.tsx:137]: maybe ideaHeader would be a better name as it contains the owner pictue, name and idea title

File: hms-app/src/components/card-details/IdeaDetails.tsx [hunk 2]
```diff
@@ -132,6 +132,212 @@ export default function IdeaDetails(props: IProps) {
...
+                      {skill.name}
+                      </Badge>
+                  </Tooltip>
+                  ))}
+              </Group>
+          </div>
+          )
+  }
+
+  const ideaSkills = ()  => {
```
Comment (LetsPoke) [inline hms-app/src/components/card-details/IdeaDetails.tsx:240]: this const feels unnessasary 
you could replace the "div" in required skills const with "card.section" and would save some lines of code

**Review by LetsPoke — CHANGES_REQUESTED** [review]
1. in the ideation portal im able to see all buttons that only the idea owner should be able to see
2. in the hackathon page I cant see any ideas
3. can you show me your prettier settings? look like we have different ones

File: hms-app/src/components/card-details/IdeaDetails.tsx [hunk 3]
```diff
@@ -462,125 +506,234 @@ export default function IdeaDetails(props: IProps) {
...
+            </Card>
+            )
+        }
     </>
-  )
+)
 }
+
+
+    // __DO NOT DELETE FOR NOW (MINIMAL CARD)__
```
Comment (LetsPoke) [inline hms-app/src/components/card-details/IdeaDetails.tsx:559]: why should we merge it with outcommented code?

**Review by reyhanrumengan — APPROVED** [review]

**Review by LetsPoke — APPROVED** [review]

**Pull request merged**